docs/reference/messaging/send.mdx

```diff
@@ -132,7 +132,7 @@ flowchart LR
       Sender -- "dispatch(...){value}" --> M_O
 
       M_O -. "fee = quoteDispatch(...)" .- R_H
-      M_O -- "postDispatch(...)<br>{fee}" --> R_H
+      M_O -- "postDispatch(...)<br>{value}" --> R_H
```

Comment:
this part is inaccurate
see
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/a82b4b4cbb14bf37be4bd60cc46d7bc0b5e6c316/solidity/contracts/Mailbox.sol#L306-L311

Reply:
Maybe the flow chart is trying to depict the happy path where docs value > docs fee. If that's the case then we can revert my change.

Reply:
yes, the diagram shows value > docs fee
value < docs fee will revert